pkg/commands/generate.go

```diff
@@ -32,31 +33,65 @@ var Generate = &cli.Command{
 			region = "us-east-1"
 		}
 
-		var o string
-		fmt.Print("Overwrite current config(~/.aws/config)[y/N]? ")
-		_, err := fmt.Scanf("%s", &o)
-		if err != nil {
-			fmt.Println("Default option")
-		}
-		switch {
-		case o == "y" || o == "Y":
-			err := aws.GenerateConfig(ssoURL, region, true)
-			if err != nil {
-				log.Fatalf("Error to generate config file: %v", err)
-			}
-		case o == "n" || o == "N":
-			err := aws.GenerateConfig(ssoURL, region, false)
-			fmt.Println("You can use this profile values to update your config file(~/.aws/config)")
+		configDir := configDirName()
+
+		if !fileExists(configDir + "/config") {
```

Comment:
nit: here we could use filepath.Join here to concatenate the path, wdyt?

Reply:
Thanks, changed!

Reply:
Perfect, thanks for the changes!